franklin/blocks/short-card/short-card.js

```diff
@@ -0,0 +1,173 @@
+import {
+  getConfig,
+  createTag,
+  loadStyle,
+  decorateLinkAnalytics,
+  decorateButtons,
+  getMetadata,
+} from '../../scripts/scripts/utils.js';
+
+const HALF = 'OneHalfCard';
+const HALF_HEIGHT = 'HalfHeightCard';
+const PRODUCT = 'ProductCard';
+const DOUBLE_WIDE = 'DoubleWideCard';
+const SHORT = 'ShortCard';
+
+const getCardType = (styles) => {
```

Comment:
Can we add JSDocs for all these functions and the static variables above? We'll need clarity going forward as to what everything does.